apps/frontend/src/pages/access-management/libs/components/group-users-table/group-users-table.tsx

```diff
@@ -48,12 +54,19 @@ const GroupUsersTable = ({
 		totalItemsCount: usersTotalCount,
 	});
 
+	const handleSearchChange = useCallback(
```

Comment:
Should we add a debounce mechanism in the onChange handler to delay search queries until the user has stopped typing for a short period?

Reply:
@Fjortis We already have debounce in Search component

Reply:
You're right, apologies for any confusion